### ISSUE TITLE
blocked?

### ISSUE THREAD

**Issue by rfecf** [carbonnetwork-dev/Carbon#7]
wheni open it, it says site cannot be reached for every thing

**Comment by Ducky443747** [comment]
yeah same for me, it was working just last friday but now I can't get it to work anymore not sure if lightspeed saw how this worked and patched it or something

**Comment by Trebuchet-Vault-Base** [comment]
me too

**Comment by manitopy-is-cool** [comment]
and me!

**Comment by doxjosh** [comment]
lightspeed patched it
